src/Services/Cda/CdaTemplateImportDispose.php

```diff
@@ -2033,14 +2033,13 @@ public function findClosestEncounter($item_date, $item_pid)
      */
     public function findClosestEncounterWithForm($item_date, $item_pid, $form = '')
     {
-        $form = add_escape_custom($form);
```

Comment:
@sjpadgett , let me know if you want me to change above to (would keep with your style of escaping variables prior):
```
$form = escape_table_name($form);
```
(and then I would remove the line below)

Reply:
No you're fine.